File: CoreTransformer/TSTTransitionGroup.vb
'************************************************************************************
' ST-Sim: A .NET class library for developing state-and-transition simulation models.
'
' Copyright © 2009-2015 ApexRMS.
'
'************************************************************************************

Friend Class TstTransitionGroup

    Private m_GroupId As Integer

    Public Sub New(ByVal groupId As Integer)
        Me.m_GroupId = groupId
    End Sub

    Public ReadOnly Property GroupId As Integer
        Get
            Return Me.m_GroupId
        End Get
    End Property

End Class

File: src/Runtime/TSTTransitionGroup.cs
// stsim: A SyncroSim Package for developing state-and-transition simulation models using ST-Sim.
// Copyright © 2007-2024 Apex Resource Management Solutions Ltd. (ApexRMS). All rights reserved.

namespace SyncroSim.STSim
{
    internal class TstTransitionGroup
    {
        private int m_TSTGroupId;

        public TstTransitionGroup(int tstGroupId)
        {
            this.m_TSTGroupId = tstGroupId;
        }

        public int TSTGroupId
        {
            get
            {
                return this.m_TSTGroupId;
            }
        }
    }
}
